cmd/crdvalidator/annotation/annotation.go

```diff
@@ -0,0 +1,7 @@
+package annotation
+
+const (
+	ValidationKey = "core.rukpak.io/safe-crd-upgrade-validation"
+	Enabled       = "true"
```

Comment:
Do we use/set this variable anywhere? It looks like we just use the Disabled key throughout this diff?

Reply:
We don't, but wanted to keep it set incase we want to add `opt-in` functionality. To your point, we can add that when we need it.